src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -226,6 +227,11 @@ def load_arguments(self, _):
             c.argument('deployment_source_branch', options_list=['--deployment-source-branch', '-b'],
                        help='the branch to deploy')
             c.argument('tags', arg_type=tags_type)
+        with self.argument_context(scope + ' config appsettings') as c:
+            c.argument('settings', nargs='+', help="space-separated app settings in a format of `<name>=<value>`")
+            c.argument('setting_names', nargs='+', help="space-separated app setting names")
+        with self.argument_context(scope + ' config appsettings list') as c:
```

Comment:
Didn't delete those lines. I moved those lines into a scope including logicapp so those params also apply to logicapp.